advanced/attributes/src/main/java/org/arakhne/afc/attrs/attr/AttributeType.java

```diff
@@ -152,53 +155,118 @@
 		NAME_RESOURCE_FILE = pName + ".types"; //$NON-NLS-1$
 	}
 
+	/** FactoryMap collection.
+	 *
+	 */
+	private static final Map<AttributeType, StringFactory> FACTORY_MAP =
```

Comment:
This attribute may be declared as SoftReference for enabling memory block collecting.